src/fiatconnect/FiatDetailsScreen.tsx

```diff
@@ -189,6 +264,11 @@ const FiatDetailsScreen = ({ route, navigation }: Props) => {
           },
           schemaCountryOverrides
         )
+      case FiatAccountSchema.IBANNumber:
+        return getIbanNumberSchema({
+          country: userCountry.countryCodeAlpha2 || 'US',
```

Comment:
What should the default country be if `countryCodeAlpha2` isn't present?

Reply:
no US banks support IBAN, so not the US

Reply:
maybe Nigeria?

Reply:
should probly log an error if `countryCodeAlpha2` is missing

Reply:
seems very likely that a bug would result-- and possibly get disguised by this default-- if that field was missing